### DISCUSSION TITLE
nxt need increase text size for the property editor and other menus in 4k monitors. difficult to read,

### DISCUSSION THREAD

**Discussion by luissjp** [nxt-dev/nxt_editor#262] [Ideas]
we can change the text size in the code editor, but In monitor that have 4k resolution text is to little to read, will be great to have a option to increase decrease text size, right now temp solution is to change second monitor to a lower resolution.

**Comment by ImLucasBrown** [comment]
Resolved in [3.14](https://github.com/nxt-dev/nxt_editor/releases/tag/editor_v3.14.0)
